CHANGELOG.md

```diff
@@ -7,122 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Code cleanup around tx logic: constants/transactions, `useTransactionDisplayData`, and `TransactionIcon` ([#14784](https://github.com/MetaMask/metamask-extension/pull/14784))
+- Removed draft transaction object from the send slide, for avoiding sync errors ([#14701](https://github.com/MetaMask/metamask-extension/pull/14701))
+- Update "secp256k1" for fast module init via lazy loading ([#14677](https://github.com/MetaMask/metamask-extension/pull/14677))
+- Update the color of the ledger alert text to adhere to design system guidelines ([#14674](https://github.com/MetaMask/metamask-extension/pull/14674))
+- Update version parsing to allow rollback release on Chrome ([#14288](https://github.com/MetaMask/metamask-extension/pull/14288))
+- Make existing component `SiteOrigin` reusable across all contexts where we surface a connected site's origin ([#14720](https://github.com/MetaMask/metamask-extension/pull/14720))
+- Swaps: handle an RPC provider delay, by returning an estimated post tx balance amount ([#14821](https://github.com/MetaMask/metamask-extension/pull/14821))
+- Swaps: reduce the default slippage from 3% to 2% ([#14863](https://github.com/MetaMask/metamask-extension/pull/14863))
+- Swaps: update STX status content page ([#14805](https://github.com/MetaMask/metamask-extension/pull/14805))
+  - Change STX status timer description dynamically based on the stxStatusDeadline param
+  - Change "Swap complete in < X:XX" to "Swap will complete in < X:XX"
+  - Change "Privately submitting the Swap..." to "Privately submitting your Swap..."
+- Swaps: migrate the estimateGas API call to getFees for STX for reducing API calls ([#14767](https://github.com/MetaMask/metamask-extension/pull/14767))
+- Swaps: disable STX if a regular tx is in progress ([#14554](https://github.com/MetaMask/metamask-extension/pull/14554))
+- Swaps: STX improvements ([#14622](https://github.com/MetaMask/metamask-extension/pull/14622))
+  - Track real fees in ETH for approval and trade txs in the Swap Completed event
+  - Use USDConversionRate for calculating estimated fees in USD
+  - Fix a condition for checking non-prod `METAMASK_ENVIRONMENT`
+  - Load `parameters.stxMaxFeeMultiplier` from network config params
+- Swaps: remove gas editing ([#14673](https://github.com/MetaMask/metamask-extension/pull/14673))
+
+### Fixed
+- Fix line-wrapping in Edit Nickname screen by decreasing the padding ([#14842](https://github.com/MetaMask/metamask-extension/pull/14842))
+- Fix permissions to https://chainid.network/chains.json by adding the domain into the manifest.json file ([#14810](https://github.com/MetaMask/metamask-extension/pull/14810))
+- Fix switching between ETH and USD, so the amount value is updated correctly ([#13827](https://github.com/MetaMask/metamask-extension/pull/13827))
+- Fix add recipient flow issue with contact list/recent recipients ([#14771](https://github.com/MetaMask/metamask-extension/pull/14771))
```

Comment:
Suggestion to change "Fix add recipient flow issue with contact list/recent recipients" to "Fix addition of 'add recipient' events to the send flow change logs so that 'contact' and 'recent' recipient are correctly distinguished"

Reply:
noted. Ty